docs/using-the-jdbc-driver/HostAvailabilityStrategy.md

```diff
@@ -0,0 +1,23 @@
+|------------------------------------------------|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Simple Host Availability Strategy              | `null` or blank      | This strategy is a simple pass through strategy. It does nothing when availability is set, and returns the raw host availability when fetched.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | 
```

Comment:
`null` or blank isn't very clear to me. What do you mean. If this is the default strategy I prefer we leave it out of the table and explicitly explain it below.

Reply:
Thats right. This is the default strategy. I'll be doing as you suggested.

Reply:
Done!